zip-1014.rst

```diff
@@ -255,6 +258,33 @@ The funds SHALL be received and administered by ZF. ZF MUST disburse them as
    an immediate family relationship with any of the above. The ZF SHALL continue
    to operate the Community Advisory Panel and SHOULD work toward making it more
    representative and independent (more on that below).
+   
+8. From 1st January 2022, a portion of the MG Slice shall be allocated to a 
+   Discretionary Budget, which may be disbursed for expenses reasonably related 
+   to the administration of Major Grants. The amount of funds allocated to the 
+   Discretionary Budget SHALL be decided by the ZF's Community Advisory Panel or 
+   successor process. Any disbursement of funds from the Discretionary Budget 
+   MUST be approved by the Major Grant Review Committee. Expenses related to the 
+   administration of Major Grants include, without limitation the following:
+   
+   * Paying third party vendor for services related to domain name registration, 
+     website design, website hosting and administration for the Major Grant 
+     Review Committee.
+   * Paying independent consultant to develop requests for proposals that align 
+     with the Major Grants program.
+   * Paying independent consultant for expert review of grant applications.
+   * Paying for sales and marketing services to promote the Major Grants 
+     program.
+   * Paying third party consultant to undertake activities that support the 
+     purpose of the Major Grants program. 
```

Comment:
Can we make this more concrete?

Reply:
Well, this is a list of examples, and is therefore _not_ intended to be exhaustive or restrictive. Making it more concrete would implicitly be restrictive.

Reply:
ok